API Blueprint Specification.md

```diff
@@ -1034,7 +1228,7 @@ Defined by the `Data Structures` keyword.
 #### Description
 This section holds arbitrary data structures definitions defined in the form of [MSON Named Types][].
 
-Data structures defined in this section **may** be used in any [Attributes section][]. Similarly, any data structures defined in a [Attributes section][] of a named [Resource Section][] **may** be used in a data structure definition.
+Data structures defined in this section **may** be used in any [Attributes section][]. Similarly, any data structures defined in a [Attributes section][] of a named [Resource section][] **may** be used in a data structure definition.
```

Comment:
As tempting as it is, please avoid doing fixes to other things in this PR – it will be easier to review and maintain. Make another PR to fix things like these.